messages/training-checkout.txt

```diff
@@ -6,17 +6,31 @@ Thanks again for taking part in Software Carpentry (SWC) and Data Carpentry (DC)
 
-2. If you want to teach DC, pick one of its lessons (possibly the one you chose for step 1, but not necessarily) and sign up on http://pad.software-carpentry.org/lesson-discussion-2016 to take part in an hour-long discussion session about it.  Similarly, if you want to teach SWC, pick one of its lessons and sign up for a discussion session about it (same Etherpad).  If there aren't upcoming slots on the Etherpad that work for you, please add yourself to the top and we'll set something up.  You must do this step separately for each Carpentry, and you're expected to read through the lesson(s) carefully *before* your session and show up with questions.  If the session moderator (an experienced instructor) feels you haven't done this, you'll be invited to come back to a future session.
+We strongly prefer contributions to the Data Carpentry lessons right now (since their material is much younger), or to our fledgling "Python for complete novices" lesson.  You can find links to the Data Carpentry lessons on http://www.datacarpentry.org/lessons/, and the Python lesson on https://github.com/swcarpentry/python-novice-gapminder.  Whatever you contribute to, and however you do it (by pull request or by adding to the Google Doc), please:
 
-3. Once you're through the discussion session(s), sign up for a 5-minute online demonstration lesson slot on http://pad.software-carpentry.org/teaching-demos (or add yourself to the top of the pad if none of the available slots works for you).  In that, we will ask you to teach a short segment from your chosen lesson - we'll pick the starting point on the day, so you will need to be familiar with the whole lesson.  If that goes well, we will send you your certificate, add you to our roster, and get you on the dance floor.
+a) Look at how the existing exercises in that lesson are formatted, and format yours in the same way.
 
-Your change can be a fix to the existing material, a new exercise, or a diagram of some kind (preferably in a vector format like SVG, but we'll take anything).  We are *not* presently accepting material with new concepts: the lessons are full to bursting as it is, and our rule is that new stuff can only be added if we can identify something to take out.  (Note: refactorings that rearrange the order of material are also welcome.)
+b) If you're submitting by PR, please include discussion in the comments on the PR (*not* in the submitted content) about the purpose of the exercise.
```

Comment:
I'd rather have some of that motivational discussion in the commit messages (which are easier to find via `git blame …` than the PR that landed the commit and then the appropriate comment in the PR discussion).